src/node_crypto.cc

```diff
@@ -544,14 +545,18 @@ void SecureContext::Initialize(Environment* env, Local<Object> target) {
   t->Set(FIXED_ONE_BYTE_STRING(env->isolate(), "kTicketKeyIVIndex"),
          Integer::NewFromUnsigned(env->isolate(), kTicketKeyIVIndex));
 
-  t->PrototypeTemplate()->SetAccessor(
+  Local<FunctionTemplate> ctx_getter_templ =
+    FunctionTemplate::New(env->isolate(),
```

Comment:
We indent by four spaces in line continuations :)

Here and also in other files.